libs/live-config/src/providers/firebaseRemoteConfig/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Provider } from "..";
+import { ConfigInfo } from "../../LiveConfig";
+import { parser } from "./parser";
+
+// refer to https://github.com/firebase/firebase-js-sdk/blob/master/packages/remote-config/src/public_types.ts#L71 for the firebase config value interface
+export interface Value {
+  asString(): string;
+  asNumber(): number;
+  asBoolean(): boolean;
+}
+
+export class FirebaseRemoteConfigProvider implements Provider {
+  public getValue: (key: string) => Value;
+
+  constructor(config: { getValue: (key: string) => Value }) {
+    this.getValue = config.getValue;
+  }
+
+  getValueBykey<K>(key: K, info: ConfigInfo) {
```

Comment:
Can we unit test main methods ? It serves as comment aswell

Reply:
we have unit test for the jsonFileReader provider getValueBykey method in "/libs/live-config/src/__tests__/LiveConfig.test.ts". 
The idea is that the LiveConfig can plug any provider as long as it implements the "getValueBykey" method.
It is not possible to write unit test for firebase remote config provider because the it queries the firebase server.

Reply:
it("should return default value if provider is undefined")
it("should return default value if parser returns null")

Reply:
fixed